### ISSUE TITLE
New anti-clumping mechanism: dimming

### ISSUE THREAD

**Issue by Anusien** [Anusien/Town-Of-Us#70]
Another idea of a way to prevent crewmates from overly clumping: Crewmate vision reduces for every player within some radius of them (probably tied to either Crewmate or impostor vision). I don’t know the best way to apply this. It might be a game setting, or a role, or a modifier that lets an impostor do this, or even a modifier that applies this while the impostor has an active kill button.

**Comment by k-electron** [comment]
its a bit forced IMO because short duration clumping is ok. Have you considered the grenadier?

**Comment by Anusien** [comment]
I agree that short duration clumping isn't a problem. I don't know the right way to do this, or even if this is the right thing to do, but it's certainly an idea that does the thing directly.

Yes, we can get the Grenadier for very little work from TOU-R: Opened #86.
